fastqpreprocessing/.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "libStatGen"]
+	path = libStatGen
+	url = git://github.com/statgen/libStatGen.git
```

Comment:
If libstatgen is a git submodule, then there's no need to check in the source code here, is there?

I'm not sure that a git submodule is the correct thing to do here. Another possible solution is checking out the repo during build. Or, if we don't need to build a custom version of it, download precompiled binaries if they exist.

Reply:
Seconding this, seems weird to copy in all of the code of another toolkit

Reply:
I removed the libStatGen code. Instead pull it from a specific release of it and apply about 10 lines of patch. Therefore, we only carry the patch file "generated by diff -u oldfile newfile"